BatchCave.py

```diff
@@ -1590,11 +1595,11 @@ def ER_ASP_WASME(self, x, name='ER-ASP-WASME'):
         return x
 
     def ER_OL_Wiley(self, x, name='ER-O/L-Wiley-InterSci'):
-        print '\nRunning change script '+ name + '\n'
+        print ('\nRunning change script '+ name + '\n')
         #breaktheMARCfile
         x = utilities.MarcEditBreakFile(x)
         #Insert 002, 003, 730, 949 before supplied 008
-        x = re.sub('(?m)^=008', r'=949  \\1$lolink$rs$t99\n=949  \\\\$a*bn=bolin;\n=003  ER-O/L-Wiley\n=002  O/L-Wiley\n=730  0\\$aWiley online library.$5OCU\n=008', x)

```

Comment:
I also didn't make this change of InterSci and InterScience ebooks. I'm really starting to think this isn't the version I downloaded. should I keep this change on my end?